poll/public/js/poll_edit.js

```diff
@@ -25,7 +25,9 @@ function PollEditUtil(runtime, element, pollType) {
                             // The degree of precision on date should be precise enough to avoid
                             // collisions in the real world.
                             var bottom = $(button_mapping[context_key]['bottomMarker']);
-                            var new_item = $(self.answerTemplate(button_mapping[context_key]['itemList']));
+                            var new_item_dict = self.extend({}, button_mapping[context_key]['template']);
+                            new_item_dict['key'] = Date.now();
+                            var new_item = $(self.answerTemplate({'items': [new_item_dict]}));
```

Comment:
@antoviaque The previous method had the Key as the date generated ahead of time in the button_mapping. But this broke things because it meant Date.now() was only evaluated once-- on the first run. Which meant that every added answer had the same key, and THERE CAN BE ONLY ONE, which is why the testers were seeing only their last answer being included alongside the defaults (which had different keys of their own)